### DISCUSSION TITLE
How to change the default View at Application Start

### DISCUSSION THREAD

**Discussion by Cetheus** [FossifyOrg/General-Discussion#471] [General]
Hi, 
is it possible to change the default View when opening the Kalendar?
When I'm starting it, I always get the Day-View.
I would like to change it to the Monthly-View or the Monthy and daily view, so I can see whats up the next days.

Thank you for the great work and the help.
Greetings Cetheus

**Comment by naveensingh** [comment]
The app remembers the last view used. If you are launching the app from widgets, please select the desired view in Settings -> View to open from event list widget.

**Reply by Cetheus** [reply]
Sadly this doesn't work. 
I checked the settings and it said, Montly-View.
But if I start it from my Widget it goes straight to Daily.
Perhaps I have to check the Widget settings from my opening widget.

**Comment by Aga-C** [comment]
Opened view depends on how you run the app. There are three cases:

- When you open the app from an icon, it always shows you the last view. Currently, there's no way to change it.
- When you open the app from the widget, it opens the view according to what you've tapped. E.g. when you tap a day in a month widget, you get a day view. The only exception from this is an event list widget, for which you can choose, in settings, what view it should open.
- When you run the app via intent (another app opening Fossify Calendar), it opens the view requested by another app.

**Comment by Cetheus** [comment]
Ok. I found it. 
I'm using Sectograph as my Widget and the default Kalender opening event gets me the day. 
I chose to open "another App" and chose "Kalender".
Now its opening in the last View I used.
Thank you for your help leading me to the solution.
Greetings Cetheus
